app/views/layout.erb

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
+    
+    <title>CRUD</title>
+
+    <!-- Bootstrap -->
+    <!-- using href="css/bootstrap.min.css" causes show.erb to ignore bootstrap -->
+    <link href="/node_modules/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet">
```

Comment:
Remove this line. This line is not necessary when we build a css file with gulp that already includes bootstrap. This might help you with setting up Gulp to do that:
https://gist.github.com/ericbarnes/ac3ae075c97c1073869c

Note that we only want to use NPM. Do not install Bower.